scripts/install_deps.py

```diff
@@ -0,0 +1,112 @@
+import os
+import platform
+import shutil
+import subprocess
+import sys
+
+from contextlib import contextmanager
+from utils import _virtualenv_enabled
+
+INSTALL_ARGS = "--no-binary :all: --no-build-isolation --no-cache-dir --no-index "
+
+
+class BadRCError(Exception):
+    pass
+
+
+class InstallationError(Exception):
+    pass
+
+
+@contextmanager
+def cd(dirname):
+    original = os.getcwd()
+    os.chdir(dirname)
+    try:
+        yield
+    finally:
+        os.chdir(original)
+
+
+def run(cmd):
```

Comment:
Can we reuse `utils.run` here instead?

Reply:
Moved cd over and am using the other util options now.